parser/pom.xml

```diff
@@ -9,6 +9,7 @@
     <artifactId>presto-coresql-parser</artifactId>
     <name>presto-coresql-parser</name>
     <properties>
+        <!--suppress UnresolvedMavenProperty -->
```

Comment:
Why do we need to add this?

Reply:
I was getting an error earlier and forgot to remove this -- I'll remove it now.